tests/robot/variables/common_variables.robot

```diff
@@ -130,6 +130,8 @@ ${RESULTS_FOLDER_SUITE}            ${RESULTS_FOLDER}/${SUITE_NAME}
 ${TEST_DATA_FOLDER}                test_data
 ${REST_CALL_SLEEP}                 0
 ${SSH_READ_DELAY}                  1
+#${CTL_TOOL}=       vppctl
+${CTL_TOOL}=       etcdctl
```

Comment:
Rename the variable to more clear name `ETCDCTL_TOOL`

Reply:
better to remove it, as it is not used anymore,  fragment left from previous version